packages/service/lib/plugins/metrics.js

```diff
@@ -7,22 +7,37 @@ const fastify = require('fastify')
 const fp = require('fastify-plugin')
 
 const metricsPlugin = fp(async function (app, opts = {}) {
-  const defaultMetrics = opts.defaultMetrics ?? { enabled: true }
   const prefix = opts.prefix ?? ''
 
+  const promClient = require('prom-client')
+
+  const register = new promClient.Registry()
+  const defaultMetrics = opts.defaultMetrics ?? { enabled: true, register }
```

Comment:
```suggestion
  const defaultMetrics = opts.defaultMetrics ?? { enabled: true }
  defaultMetrics.register = register 
```

You want to set the register anyway.

Reply:
@ivan-tymoshenko  Unfortunately, we need to have it otherwise it not used for default metrics, see:

https://github.com/SkeLLLa/fastify-metrics/blob/master/src/fastify-metrics.ts#L249

If `register` is not set in options, it uses the global one. See: https://github.com/siimon/prom-client?tab=readme-ov-file#default-metrics